src/wallet.rs

```diff
@@ -461,7 +461,7 @@ fn format_gdk_tx(txdesc: &Value, tx: Transaction) -> Result<Value, Error> {
 fn parse_outs(details: &Value) -> Result<HashMap<String, f64>, Error> {
     debug!("parse_addresses {:?}", details);
 
-    Ok(details["addressees"]
+    Ok(details["addresses"]
```

Comment:
This is not a typo, should actually be `addressees`. I initially thought it should be `addresses` too, but the Green apps did not like it. @greenaddress was saying that its intentionally spelled like that, iirc as in "the people being addressed", like "payees".

Reply:
Hmm, okay. But then something else is up because with addresses, it works and with addressees it doesn't. Let me change it back and grep for `"addresses"` somewhere else.

Reply:
Ok fixed it. The tests were just also saying `"addresses"`. Changed that and all the tests still pass.
The build times out on the `ndk.sh` execution.